sdk/component/runner.go

```diff
@@ -38,12 +36,14 @@ func newRunner(opts ...RunnerOption) (*runner, error) {
 		}
 	}
 
-	if r.config.Storer == nil {
-		storer, err := localstore.NewManager()
-		if err != nil {
-			return nil, errors.Errorf("could not create default storer: %w", err)
+	if !r.config.StoreConfig.DisableStoreValidation {
+		if r.config.StoreConfig.Storer == nil {
```

Comment:
There's no need for nesting here, but it might benefit from a comment that makes it easier to read. 
E.g. `//setup store if setting is enabled and we don't have a store yet`

Reply:
Done